test/Conversion/ExportVerilog/instance-choice.mlir

```diff
@@ -0,0 +1,40 @@
+// RUN: circt-opt %s -export-split-verilog='dir-name=%t.dir'
+// RUN: cat %t.dir/top.sv | FileCheck %s
```

Comment:
Single file emission would keep this to one line and avoid any output (or Windows issues).